traitcuration/traits/datasources/zooma.py

```diff
@@ -98,9 +98,9 @@ def create_mapping_suggestion(trait, term):
     """
     Creates a mapping suggestion in the app's database, if it doesn't exist already.
     """
-    zooma = User.objects.filter(username="ZOOMA").first()
+    zooma = User.objects.filter(first_name="ZOOMA").first()
```

Comment:
Filtering by the name seems a bit wrong. I'm not saying we are likely to encounter someone with the first name “ZOOMA”, but maybe it would be better to filter by email

Reply:
I actually added that in PR #63 so should I leave it as it is and wait for both PRs to merge into master?

Reply:
Yes, if it's already planned to be fixed in a subsequent PR, then that's fine, no need to backport the fix here